src/Cake.Common/Tools/NuGet/Update/NuGetUpdateSettings.cs

```diff
@@ -53,5 +54,11 @@ public sealed class NuGetUpdateSettings : ToolSettings
         /// </summary>
         /// <value>The version of MSBuild to be used with this command.</value>
         public NuGetMSBuildVersion? MSBuildVersion { get; set; }
+
+        /// <summary>
+        /// Gets or sets package version to be used with this command.
+        /// </summary>
+        /// <value>The package version to be used with this command.</value>
+        public Version Version { get; set; }
```

Comment:
Don't think `System.Version` is the best type here as it don't supports SemVer, which is required if one would like to use a prerelease version.

Reply:
Along with this, we should also probably add `PreRelease` argument as well as it goes hand in hand with the `Version` argument.

Reply:
ah. Thanks @mholo65 for the insight. Changed `Version` prop to string to accommodate SemVer. The `PreRelease` option is already there.